package.json

```diff
@@ -8,7 +8,8 @@
     "test": "karma start --single-run --no-auto-watch && npm run eslint",
     "test-watch": "karma start --auto-watch --no-single-run",
     "release": "node tasks/release.js",
-    "i18n": "npm run dist && node tasks/i18n"
+    "i18n": "npm run dist && node tasks/i18n",
+	"update-deps": "rm npm-shrinkwrap.json && rm -rf node_modules && npm install --no-optional && npm install --no-optional && npm prune && npm shrinkwrap --dev"
```

Comment:
Yeah, this is that `wp-calypso` does. I really don't want to learn why do they have to `npm install` twice, I prefer to live in ignorance.

Note that this is only needed to regenerate the `shrinkwrap` file, you **shouldn't** need to do this almost ever.